PZ/templates/measurements.html

```diff
@@ -1,5 +1,7 @@
 {% extends "base.html" %}
 {% block content %}
+    <p align="right"><font color="white">Logged as {{ full_name }} </font></p>
```

Comment:
multiple times use the same code. the better way is create separate base html file. 
http://stackoverflow.com/questions/31456576/django-footer-and-header-on-each-page-with-extends